examples/cluster/azure/terraform.template.tfvars

```diff
@@ -29,3 +29,35 @@ support = {
   plan     = "basic"
   timezone = "PT"
 }
+
+
+# Example for Ultra disk type Azure Cluster:
```

Comment:
Can we make this a different example file? so that we don't need to comment it and the file will be usable as-is.